frontend/vite.config.ts

```diff
@@ -1,7 +1,13 @@
 import { defineConfig } from 'vite';
 import react from '@vitejs/plugin-react-swc';
+import path from 'path';
 
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
+  resolve: {
+    alias: {
+      '@': path.resolve(__dirname, './src'),
+    },
```

Comment:
Even this, what is the use case for this

Reply:
It is commonly used to avoid long relative import paths in code.
`import { cn } from "@/lib/utils`  - used here
For instance, instead of importing a file like `../../components/SomeComponent`, you can simply use `@/components/SomeComponent`.
If don't want this extra setup we can use as `../../components/SomeComponent`.

Reply:
![image](https://github.com/krishnaacharyaa/wanderlust/assets/116620586/8dd9a272-4036-4397-88b3-1f95e0b210e0)

Then we would have to change everywhere to maintain consistency.

Reply:
This alias is actually need for shadcn's components. So we need to keep this.
![image](https://github.com/krishnaacharyaa/wanderlust/assets/112842857/635b6f50-51c0-4bdf-a509-6b416aab8b7f)

Reply:
Yes please do, lets make the best use of the same everywhere and maintain readability and consistency

Reply:
Hii @krishnaacharyaa, I have updated import everywhere to maintain readability and consistency :)